### PR TITLE
Code is data

### PR DESCRIPTION
**Pull request by minrk**
according to review, we should cover the code produced in our data management plan

### CODE REVIEW

File: concept.tex [hunk 1]
```diff
@@ -597,6 +597,16 @@ \subsubsection{Research Data Management and management of other outputs}
...
 
+\noindent \textbf{Software} \\
+
+Most research data form \TheProject will take the form of code.
+All code produced by \TheProject will be developed publicly,
+licensed under OSI-approved Open Source licenses,
+and hosted on public repositories, such as GitHub.
+Milestone achievements of the software may be archived
+in separate repositories such as Zenodo,
+e.g. to coincide with publications.
```
Comment (annefou) [inline concept.tex]: There are some repetitions with "1.2.12 Open science practices and implementation". Should "Open science practices and implementation" section be moved here?

Comment (minrk) [reply concept.tex]: I don't know. They seem to ask for huge amounts of redundancy. But the DMP part is more about storage/preservation (i.e. hosted on gihtub), whereas 1.2.12 is more about practices/licenses (open source, developed in public).

Comment (minrk) [reply concept.tex]: Could try to be more brief here and just mention public hosting and a reference to the open practices.

Comment (annefou) [reply concept.tex]: You're right. Maybe make a reference to the previous section? 

You also have a typo: Most research data form \TheProject   --> Most research data from \TheProject 

Such as:

Most research data from \TheProject will take the form of code and will be developed publicly (see section 1.2.12).
Milestone achievements of the software may be archived
in separate repositories such as Zenodo,
e.g. to coincide with publications.

**Comment by fangohr** [comment]
This looks good to me now. I'll merge it later if there is no further input / feedback.

**Review by annefou — APPROVED** [review]

**Pull request merged**